pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -689,6 +690,67 @@ func TestJavaInjection(t *testing.T) {
 				os.Remove(testfile)
 			},
 		},
+		{
+			// Test the java jdk client https request is working
+			name: "java_jdk_client_httpbin",
+			context: testContext{
+				extras: make(map[string]interface{}),
+			},
+			preTracerSetup: func(t *testing.T, ctx testContext) {
+				o, err := exec.Command("java", "-version").CombinedOutput()
+				if err != nil {
+					t.Fatalf("java -version failed to exec %v", err)
+				}
+				t.Skipf("unsure how to parse different java -version %s", string(o))
+				cfg.JavaDir = legacyJavaDir
+			},
+			postTracerSetup: func(t *testing.T, ctx testContext) {
+				javatestutil.RunJavaHost(t, "Wget", []string{"https://httpbin.org/anything/java-tls-request"}, regexp.MustCompile("Response code = .*"))
+			},
+			validation: func(t *testing.T, ctx testContext, tr *Tracer) {
+				time.Sleep(time.Second)
```

Comment:
Is the additional second wait necessary? Could you adjust the maximum wait in the `require.Eventually`? As is, this waits for 2-4 seconds. Is 1-4 seconds ok? That way the tests can complete faster if conditions are met.

Reply:
I removed the pre-sleep, (copy/paste history from previous test)

Reply:
The next PR will change this test and will not do fatal but return a boolean